tests/spec/filer.buffer.spec.js

```diff
@@ -12,38 +14,38 @@ describe('Filer.Buffer', function() {
   });
 
   it('should support .isBuffer()', function() {
-    var buf = Buffer.alloc(0);
+    let buf = Buffer.alloc(0);
     expect(Buffer.isBuffer(buf)).to.be.true;
   });
 
   describe('Deprecation checks - constructor vs. class method init', function() {
 
     it('should allow new Buffer(array)', function() {
-      var arr = [1, 2, 3];
-      var buf1 = new Buffer(arr);
-      var buf2 = new Buffer.from(arr);
+      let arr = [1, 2, 3];
+      let buf1 = new Buffer(arr);
+      let buf2 = new Buffer.from(arr);
       expect(buf1).to.deep.equal(buf2);
     });
 
     it('should allow new Buffer(ArrayBuffer)', function() {
-      var arrayBuffer = (new Uint8Array([1, 2, 3])).buffer;
-      var buf1 = new Buffer(arrayBuffer);
-      var buf2 = Buffer.from(arrayBuffer);
+      let arrayBuffer = (new Uint8Array([1, 2, 3])).buffer;
+      let buf1 = new Buffer(arrayBuffer);
+      let buf2 = Buffer.from(arrayBuffer);
       expect(buf1).to.deep.equal(buf2);
     });
 
     it('should allow new Buffer(ArrayBuffer)', function() {
-      var buffer = new Buffer.from([1, 2, 3]);
-      var buf1 = new Buffer(buffer);
-      var buf2 = Buffer.from(buffer);
+      let buffer = new Buffer.from([1, 2, 3]);
+      let buf1 = new Buffer(buffer);
+      let buf2 = Buffer.from(buffer);
       expect(buf1).to.deep.equal(buf2);
     });
 
     it('should allow new Buffer(string)', function() {
-      var s = 'Hello World';
-      var buf1 = new Buffer(s);
-      var buf2 = Buffer.from(s);
+      let s = 'Hello World';
+      let buf1 = new Buffer(s);
+      let buf2 = Buffer.from(s);
       expect(buf1).to.deep.equal(buf2);
     });
   });
-});
+});
```

Comment:
In general we try to avoid this (i.e., not having a newline at the end of a file).  The reason we do this is so that if this file is concatenated with another file, the last line of this file, and the first line of some other file, don't get joined and cause some weird error.

Always include a newline at the end of all your files.